packages/synapse-react-client/src/lib/containers/home_page/featured-data/FacetPlotsCard.tsx

```diff
@@ -145,7 +150,12 @@ const FacetPlotsCard: React.FunctionComponent<FacetPlotsCardProps> = ({
         >
           <span className="FacetPlotsCard__title">{cardTitle}</span>
           {description && (
-            <span className="FacetPlotsCard__description">{description}</span>
+            <span className="FacetPlotsCard__description">
+              <ShowMore
```

Comment:
Wrap description in a ShowMore because the descriptions are so long in the EL portal